index.html

```diff
@@ -6,49 +6,208 @@
+            <hr class="horizontal-seperator">
+        </div>
+        <div class="work-card-container">
+            <div class="img-one img"></div>
+            <h2 class="card-heading">Multi-Post Stories</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <div class="skills">
+                <input type="button" value="CSS">
+                <input type="button" value="HTML">
+                <input type="button" value="Bootstrap">
+                <input type="button" value="Ruby">
+            </div>
+            <div class="projects">
+                <input type="button" value="See project">
+            </div>
+            <div class="img-two remaining-img">
+                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                    industry's standard.
+                </p>
+                <div class="skills-details">
+                    <input type="button" value="HTML">
+                    <input type="button" value="Bootstrap">
+                    <input type="button" value="Ruby">
+                </div>
+                <div class="projects-details">
+                    <input type="button" value="See project">
+                </div>
+            </div>
+            <div class="img-three remaining-img">
+                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                    industry's standard.
+                </p>
+                <div class="skills-details">
+                    <input type="button" value="HTML">
+                    <input type="button" value="Bootstrap">
+                    <input type="button" value="Ruby">
+                </div>
+                <div class="projects-details">
+                    <input type="button" value="See project">
+                </div>
+            </div>
+            <div class="img-four remaining-img">
+                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                    industry's standard.
+                </p>
+                <div class="skills-details">
+                    <input type="button" value="HTML">
+                    <input type="button" value="Bootstrap">
+                    <input type="button" value="Ruby">
+                </div>
+                <div class="projects-details">
+                    <input type="button" value="See project">
+                </div>
```

Comment:
- [ ] kindly consider adding the button styling: Active, Hover, and Pressed to improve the user experience